janitor/functions/pivot.py

```diff
@@ -282,7 +336,7 @@ def pivot_wider(
     df = df.copy()
 
     return _computations_pivot_wider(
-        df,
```

Comment:
Ooh, I'm a little curious why this copy is needed?

Reply:
To avoid mutating the dataframe

Reply:
Sure, I get that. But isn't it already copied 3 lines above?

Reply:
ahhh ... error on my part